bevrand.componenttests/tests/test_highscore_api.py

```diff
@@ -1,41 +1,49 @@
-from tests import test_setup_fixture
+from time import sleep
+
 from helpers.random_name_generator import HelperClass
+from tests import test_setup_fixture
 
 
 class HighScoreApiTests(test_setup_fixture.TestFixture):
-
     def test_ping_returns_200(self):
         split_url = "/".join(self.highscore_url.split("/", 3)[:3])
-        sut = split_url + '/ping'
+        sut = split_url + "/ping"
         response = self.get_without_auth_header(sut)
         self.assertEqual(200, response.status_code)
 
-    def test_should_get_a_404_for_a_user_and_playlist_that_does_not_exist(self):
+    def test_should_get_a_404_for_a_user_and_playlist_that_does_not_exist(
+            self):
         playlist = HelperClass.random_word_letters_only(25)
         user = HelperClass.random_word_letters_only(25)
-        sut = self.highscore_url + f'{user}/{playlist}/'
+        sut = self.highscore_url + f"{user}/{playlist}/"
         resp = self.get_without_auth_header(sut)
         self.assertEqual(404, resp.status_code)
 
     def test_should_get_a_404_for_a_playlist_that_does_not_exist(self):
         playlist = HelperClass.random_word_letters_only(25)
         user = HelperClass.random_word_letters_only(25)
-        sut = self.highscore_url + f'/{user}/{playlist}/'
+        sut = self.highscore_url + f"/{user}/{playlist}/"
         resp = self.get_without_auth_header(sut)
         self.assertEqual(404, resp.status_code)
 
     def test_should_get_user_lists_when_randomizing(self):
-        sut = self.randomizer_url + '/v1/randomize'
+        sut = self.randomizer_url + "/v1/randomize"
         body = self.test_randomize_body
-        self.post_without_auth_header(sut, body)
+        for x in range(3):
+            self.post_without_auth_header(sut, body)
```

Comment:
Should we break the for if the right response was found, and do we need a sleep in between retries?